helpers/control_node_setup.sh

```diff
@@ -0,0 +1,38 @@
+# Instructions to run this script. PLEASE READ!
+# 1. Update USERNAME, PASSWORD - only if RHEL control node is used
+# 2. Update RHEL_MAJOR_VERSION - as per the instructions given
+# 3. Make sure to run this script as root, in this project's root folder
+
+export USERNAME=updateme
+export PASSWORD=updateme
+# RHEL_MAJOR_VERSION should be as follows:
+# RHEL6: 6
+# RHEL7: 7
+# Fedora: Comment or remove the below line
+export RHEL_MAJOR_VERSION=7
+
+if [ -z "$RHEL_MAJOR_VERSION" ]; then
+  dnf install -y ansible
+else
+  # Register and subscribe
+  echo "Register and subscribe the node. This is done to install few packages."
+  subscription-manager register --username="$USERNAME" --password="$PASSWORD" --autosubscribe
+
+  set -x
+  # Enable epel repo and install Ansible
+  wget https://dl.fedoraproject.org/pub/epel/epel-release-latest-"$RHEL_MAJOR_VERSION".noarch.rpm
+  yum localinstall epel-release-latest-"$RHEL_MAJOR_VERSION".noarch.rpm
```

Comment:
Is there a reason you just don't do:

```
yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-"$RHEL_MAJOR_VERSION".noarch.rpm
```

This will work in RHEL6 and RHEL7